R/pkg/R/serialize.R

```diff
@@ -71,18 +69,19 @@ writeObject <- function(con, object, writeType = TRUE) {
   switch(serdeType,
          NULL = writeVoid(con),
          integer = writeInt(con, object),
+         glue =,
```

Comment:
can this be done without hardcoding "glue" since it's not an explicit dependency or a R base type?

Reply:
That's what would be accomplished by moving this logic to S3 or S4 -- `glue` would end up being handled by `writeType.character` (etc)

Reply:
ok that's fair. I'm ok with what you have now. could you open a jira about s3 writeType?

Reply:
Yes. I'm about halfway done working on the switch to S3 also.

Reply:
Filed: https://issues.apache.org/jira/browse/SPARK-28070